definitions.py

```diff
@@ -33,6 +33,7 @@ class ROI:
             ID     = "ID"
             COORDS = "Coordinates"
             SHAPE  = "Shape"
+            COLOR  = "Color"
```

Comment:
In Qt 'hdf5_definitions.h' file, `COLOR` is defined as dataset attribute, I'd do the same here